dags/trigger_dag.py

```diff
@@ -0,0 +1,85 @@
+"""
+DAG identify zip files to process from s3 bucket and trigger dag for each zip file
```

Comment:
(the Python module) could be a more generic name e.g. `watch_dag` or `s3_watch_dag`, but it's more complicated than this